genesyscloud/idp_adfs/resource_genesyscloud_idp_adfs_schema.go

```diff
@@ -75,7 +75,7 @@ func ResourceIdpAdfs() *schema.Resource {
 				Description:  `Valid values: HTTP Redirect, HTTP Post`,
 				Optional:     true,
 				Type:         schema.TypeString,
-				ValidateFunc: validation.StringInSlice([]string{`HTTP Redirect`, `HTTP Post`}, false),
+				ValidateFunc: validation.StringInSlice([]string{`urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect`, `urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST`}, false),
```

Comment:
I would just remove the ValidateFunc altogether. There might be customers that are using "HTTP Redirect" or "HTTP Post" and this would be a breaking change. You can also just remove the description since the API doesn't provide one and the current one won't apply anymore.